investd/reports/overview.py

```diff
@@ -27,13 +28,20 @@
 df_tx = load_transactions()
 df_tx = df_tx[df_tx["timestamp"] <= now]
 
-# %% [markdown]
-# ### Invested amount
+# %%
+
+df_quotes = load_quotes()
+df_portfolio = views.portfolio_value(
+    df_tx, df_quotes, at_date=date(2022, 12, 30)
+).round(2)
+
+display(df_portfolio)
 
 # %%
 Markdown(
     f"""
-{views.total_invested_ref_currency(df_tx):.2f} {INVESTD_REF_CURRENCY}
+Portfolio value at date {INVESTD_REF_CURRENCY}: {df_portfolio[f"Amount at date {INVESTD_REF_CURRENCY}"].sum()}
```

Comment:
maybe print the date as well?

Reply:
Check if with the report date it's clear now